<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelDialogUI</class>
 <widget class="QWidget" name="ChannelDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>64</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="KPasswordLineEdit" name="m_passwordEdit">
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="m_channelEdit">
     <property name="maxLength">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passwordLbl">
     <property name="text">
      <string>Pass&amp;word:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>m_passwordEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="channelLbl">
     <property name="text">
      <string>C&amp;hannel:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>m_channelEdit</cstring>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>m_passwordEdit</zorder>
  <zorder>m_channelEdit</zorder>
  <zorder>passwordLbl</zorder>
  <zorder>channelLbl</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPasswordLineEdit</class>
   <extends>QWidget</extends>
   <header>kpasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
